src/main/java/datawave/microservice/authorization/util/AuthorizationsUtil.java

```diff
@@ -0,0 +1,177 @@
+package datawave.microservice.authorization.util;
+
+import com.google.common.base.Splitter;
+import com.google.common.collect.Iterables;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import datawave.accumulo.util.security.UserAuthFunctions;
+import datawave.microservice.authorization.user.DatawaveUserDetails;
+import datawave.security.authorization.DatawaveUser;
+import datawave.security.util.AuthorizationsMinimizer;
+import org.apache.accumulo.core.security.Authorizations;
+import org.apache.commons.lang3.StringUtils;
+
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.LinkedHashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+
+public class AuthorizationsUtil {
+    
+    public static Authorizations union(Iterable<byte[]> authorizations1, Iterable<byte[]> authorizations2) {
+        LinkedList<byte[]> aggregatedAuthorizations = Lists.newLinkedList();
+        addTo(aggregatedAuthorizations, authorizations1);
+        addTo(aggregatedAuthorizations, authorizations2);
+        return new Authorizations(aggregatedAuthorizations);
```

Comment:
Is it worth using a set to deduplicate?

Reply:
This is the microservices equivalent of thte AuthorizationsUtil that can be found in datawave-ws-common-util in the main datawave project. The goal was to keep the logic the same, but switch out the DatawavePrincipal for ProxiedUserDetails.  I would prefer to leave this as-is since we have been running with this for quite some time.  Thoughts?

Reply:
ok, thanks

Reply:
i moved the implementation here, and had the datawave-ws-common-util version utilize the logic in this class.